### DISCUSSION TITLE
I need to make multiselect working like select2, but without jquery ?

### DISCUSSION THREAD

**Discussion by sergeynilov** [livewire/livewire#8135] [1. Help] [answered]
In laravel 10/ livewire 3/ tailwindcss 3 app I need to make multiselect working like select2, but without jquery used,
whuch is required by  select2. Are there some decisions ?

**Comment by lautaropiatti** [comment, marked as answer]
I found myself having the same requirement that you do. This video that explains how to use Select2 with Livewire 3 has helped me to understand every step needed to make this work with ChoicesJS, which doesn't need to use jQuery.
https://www.youtube.com/watch?v=vBvW2ygSEoE

You just need to use the ChoicesJS "change" event instead of Select2 equivalent, the rest is pretty much the same.
https://github.com/Choices-js/Choices?tab=readme-ov-file#events
Like so:

```
choices.passedElement.element.addEventListener('change', () => {
  // Choices getValue(true) returns only values rather than value objects
  // $set() third param determines if requests are sent to the backend every time ... true|false
  $wire.$set('selectedTags', choices.getValue(true), false);
});
```

This old tutorial doesn't seem to work anymore with Livewire 3.
https://forum.laravel-livewire.com/t/select-and-select-multi/1120

I know this is an old thread but I wanted to shout out the guy that made the tutorial. Also I do hope that this help someone else in the future 🙂
